### PR TITLE
fix: update README

### PR DESCRIPTION
**Pull request by alissa-tung**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,13 +240,12 @@ func main() {
...
+		if res.Err != nil {
+			log.Printf("Fetch error: %s\n", res.Err)
 			continue
 		}
 
-		for _, record := range receivedRecords {
+		for _, record := range res.Result {
 			rid := record.GetRecordId()
 			log.Printf("receive recordId: %s\n", rid.String())
 			fetchRes = append(fetchRes, rid)
```
Comment (YangKian) [inline README.md:251]: should send `Ack` for each record, See https://github.com/hstreamdb/hstreamdb-go/blob/7b3daf1440acb7b7f3eb5ca4512f6831cebca1e7/integraion_test/subscription_test.go#L137-L143

**Review by YangKian — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alissa-tung) [reply README.md:251]: thx, fixed

File: README.md [hunk 2]
```diff
@@ -240,24 +240,20 @@ func main() {
...
+      record.Ack()
```
Comment (YangKian) [inline README.md:252]: Indentation error, need to align code manually

**Review by YangKian — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alissa-tung) [reply README.md]: fixed, my bad vscode auto save setting

**Review by YangKian — APPROVED** [review]

**Pull request merged**